.github/workflows/e2e_test.yaml

```diff
@@ -96,6 +96,17 @@ jobs:
       - name: Install GitHub Cli
         run: which gh || sudo apt install gh -y
 
+      - name: Check rate limit
+        env:
+          GH_TOKEN: ${{ (matrix.event.name == 'issues' || matrix.event.name == 'schedule') && secrets.E2E_TESTING_TOKEN || secrets.GITHUB_TOKEN }}
```

Comment:
Im wondering if the GH_TOKEN should come before E2E_TESTING_TOKEN, so that it gets prioritised?

Reply:
Can you explain this? This code should ensure that the `E2E_TESTING_TOKEN` is selected for `issues` and `schedule` and otherwise the `GITHUB_TOKEN`.

Reply:
Ah gotcha, thanks!